yoyodyne/data/tsv.py

```diff
@@ -0,0 +1,175 @@
+"""TSV parsing.
+
+The TsvParser yield string tuples from TSV files using 1-based indexing.
+
+The CellParser converts between raw strings ("cells") and lists of string
+symbols.
+"""
+
+import csv
+import dataclasses
+from typing import Iterator, List, Tuple
+
+from .. import defaults, util
+
+
+class Error(Exception):
+    """Module-specific exception."""
+
+    pass
+
+
+@dataclasses.dataclass
+class TsvParser:
```

Comment:
for separate PR, but i'm wondering how trivial it would be to also pass `json` now we're including other dataprocessing libraries like `csv`